src/python/zquantum/core/hamiltonian.py

```diff
@@ -201,49 +216,18 @@ def get_expectation_values_from_rdms(
     reordered_qubitoperator = QubitOperator()
     for term, coefficient in terms_iterator:
         reordered_qubitoperator += QubitOperator(term, coefficient)
-
     expectations_packed = interactionrdm.get_qubit_expectations(reordered_qubitoperator)
 
-    if () in expectations_packed.terms:
-        del expectations_packed.terms[
-            ()
-        ]  # Expectation of the constant term is excluded from expectation values
-
```

Comment:
I see a potential issue here: now the expectation values returned by this function contain the constant term, while `compute_group_variance` expects:
- ExpectationValues without the constant term
- QubitOperator with the constant term (that gets removed)